documentation/user_docs/docs/Advanced_Multifit.rst

```diff
@@ -0,0 +1,74 @@
+#################
+Advanced Multifit
+#################
+
+This document describes how to set up an object to work with Multifit specifications
```

Comment:
I am not clear if this document carries on from another document called "Basic Multifit" or similar. I have a feeling that I am expected to know something but I am not clear what. It may be good to itemise what the reader is expected to known.

Assuming that this does follow on from "Multifit.rst", something like. The page "Multifit" describes how to use multifit. Here we look at how to construct a fittable object so that multifit can use it.

When you say "an object" do you mean something like a sqw or dnd object? Why would I want to set them up?- I thought they were ready.